microsoft-edge/devtools-guide-chromium/recorder/index.md

```diff
@@ -0,0 +1,304 @@
+
+   ![The finalized recording and the Replay button](./index-images/final-recording.png)
+
+
+<!-- ====================================================================== -->
+## Record other user flows
+
+To record a new user flow, once you've stopped the recording of the previous user flow, click the **Create a new recording** (![Create a new recording button](./index-images/new-recording-icon.png)) button in the top-left corner of the **Recorder** tool, and then repeat the steps from [Record a user flow](#record-a-user-flow):
+
+![The Create a new recording button](./index-images/create-new-recording.png)
+
+
+<!-- ====================================================================== -->
+## Replay a user flow
+
+You can replay user flows either from the list of recorded user flows, or when viewing the details of a given user flow.
+
+* To view the list of recorded user flows, click the dropdown menu that's in the **Recorder** tool toolbar, at the top, and select **2 recording(s)**. The label might show a different number of recordings, depending on how many recordings you've made.
+* To view the details of a given user flow, click the dropdown menu that's in the **Recorder** tool toolbar, at the top, and click the name of the user flow you want to view.
+
+
+<!-- ------------------------------ -->
+#### Replay a user flow from the list of recordings
+
+1. Click **Play recording** next to the recording that you want to replay:
+
+   ![The list of recordings, and the play button next to each recording](./index-images/play-from-list.png)
+
+1. Wait while the **Recorder** tool replays the previously recorded user interactions. An animation of the steps that are being replayed appears in the **Recorder** tool:
+
+   ![The replayed interactions displayed as an animation in the Recorder tool](./index-images/replayed-interactions-animation.png)
+
+   When the replay is complete, the animation stops, and the **Recorder** tool displays the recording's steps.
+
+
+<!-- ------------------------------ -->
+#### Replay a user flow from a recording
+
+1. Click **Replay** in the top-right corner of the **Recorder** tool:
+
+   ![The Replay button on a recorded user flow details page](./index-images/play-from-recording.png)
+
+1. Wait while the **Recorder** tool replays the previously recorded user interactions. An animation of the steps that are being replayed appears in the **Recorder** tool:
+
+   ![The replayed interactions displayed as an animation in the Recorder tool](./index-images/replayed-interactions-animation.png)
+
+   When the replay is complete, the animation stops, and the **Recorder** tool displays the recording's steps.
+
+
+<!-- ------------------------------ -->
+#### Slowly replay a user flow to investigate issues
+
+To see what's happening in the rendered page while the user interactions are being replayed, slow down the replay speed. By default, the **Recorder** tool replays user flows as fast as it can. To slow down the replay speed:
```

Comment:
```suggestion
To see what's happening in the rendered page while the user interactions are being replayed, slow down the replay speed. By default, the **Recorder** tool replays user flows as fast as it can.

 To slow down the replay speed:
```
lead-in for list of steps is left-justified